<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>491</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Editar Partido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar Partido</string>
    </property>
   </widget>
   <widget class="QPushButton" name="golesTotalesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Goles a favor y en contra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>El equipo con m√°s goles en todas las competiciones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="promedioGolesButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio de goles a favor y encontra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Victorias y derrotas por equipo en competicion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha con mas y menos goles de un equipo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="top5Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>151</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>71</height>
     </size>
    </property>
    <property name="font">
     <font>
      <hintingpreference>PreferFullHinting</hintingpreference>
     </font>
    </property>
    <property name="text">
     <string>5 partidos con mas cantidad de goles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Equipo con menos goles en todas las competiciones</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>975</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>top5Button</tabstop>
  <tabstop>editButton</tabstop>
  <tabstop>deleteButton</tabstop>
  <tabstop>golesTotalesButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>promedioGolesButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>pushButton_7</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
